packages/schemas/alterations/next-1718175155-custom-ui-assets.ts

```diff
@@ -0,0 +1,65 @@
+import { type CommonQueryMethods, sql } from '@silverhand/slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const getId = (value: string) => sql.identifier([value]);
+
+const getDatabaseName = async (pool: CommonQueryMethods) => {
+  const { currentDatabase } = await pool.one<{ currentDatabase: string }>(sql`
+    select current_database();
+  `);
+
+  return currentDatabase.replaceAll('-', '_');
+};
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    const database = await getDatabaseName(pool);
+    const baseRoleId = getId(`logto_tenant_${database}`);
+
+    await pool.query(sql`
+      create table custom_ui_assets (
+        tenant_id varchar(21) not null
+          references tenants (id) on update cascade on delete cascade,
+        id varchar(21) not null,
+        files jsonb /* @use UploadFiles */ not null default '[]'::jsonb,
+        created_at timestamptz not null default(now()),
+        updated_at timestamptz not null default(now()),
+        primary key (id)
+      );
+      create index custom_ui_assets__id
+        on custom_ui_assets (tenant_id, id);
+
+      create trigger set_tenant_id before insert on custom_ui_assets
+        for each row execute procedure set_tenant_id();
+
+      alter table custom_ui_assets enable row level security;
```

Comment:
We have a function `enableRls` which can simplify the process

Reply:
Script updated. No more "custom-ui-assets" table needed